spec/api/entities/search_result_spec.rb

```diff
@@ -68,7 +68,8 @@
           cav_warning: 0,
           supplementary: 0,
           agfs_hardship: 0,
-          lgfs_hardship: 0
+          lgfs_hardship: 0,
+          scheme_12_warning: 0
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r476385484)

Reply:
👍 have amended naming to scheme_12_fees_warning